README.md

```diff
@@ -15,17 +18,24 @@ If you experience any issues with the older versions, you are most welcome to co
 
 ## Status
 
-Both the OPTIMADE specification and this repository are **under development**.
+Whilst v1.0 of the OPTIMADE specification has been released, this repository is **under development**.
+Outstanding features required for compliance with OPTIMADE v1.0 can be tracked with the OPTIMADE v1.0 label on [GitHub](https://github.com/Materials-Consortia/optimade-python-tools/issues?q=is%3Aopen+is%3Aissue+label%3A%22OPTIMADE+v1.0%22), which can be further filtered by backend.
+The package changelog can be found in [CHANGELOG.md](changelog.md)
```

Comment:
It doesn't make sense to write the filename when coming from the online documentation.
This should be fine as a compromise?
```suggestion
The package changelog can be found in [the changelog](CHANGELOG.md)
```